api/src/pbtar_api/app.py

```diff
@@ -0,0 +1,50 @@
+from fastapi import FastAPI
+from fastapi.responses import RedirectResponse
+from fastapi.middleware.cors import CORSMiddleware
+
+from .routers.health import health_router
+from .routers.endpoints import endpoints
```

Comment:
like/prefer the relative module syntax 👍🏻

Reply:
That's interesting!